foreman/data_refinery_foreman/foreman/main.py

```diff
@@ -1381,14 +1381,15 @@ def monitor_jobs():
                 logger.error("Caught exception in %s: ", function.__name__)
                 traceback.print_exc(chain=False)
 
-        if timezone.now() - last_janitorial_time > JANITOR_DISPATCH_TIME:
-            send_janitor_jobs()
-            cleanup_the_queue()
-            last_janitorial_time = timezone.now()
-
-        if timezone.now() - last_dbclean_time > DBCLEAN_TIME:
-            clean_database()
-            last_dbclean_time = timezone.now()
+        if not settings.RUNNING_IN_CLOUD:
```

Comment:
We want to run janitor and dbclean jobs in the cloud and not locally. 

```suggestion
        if settings.RUNNING_IN_CLOUD:
```